api/models/v2/application.go

```diff
@@ -18,3 +19,7 @@ type NavigationMenuItem struct {
 	Label       string `json:"label"`
 	Destination string `json:"destination"`
 }
+
+type StreamlitPlaceholderPageConfig struct {
```

Comment:
nit: PlaceholderPageConfig. CMIIW this is basically the url for iframe right? and the implementation of the associated page doesn't necessary streamlit, it can be used for other?

Reply:
Yeah it is basically for an iframe. Oh yes good point I can totally do away with all the references to 'Streamlit' in the code. I'll remove all of them; thanks for bringing this up!